com.unity.formats.alembic/Runtime/Scripts/Misc/Utils.cs

```diff
@@ -0,0 +1,32 @@
+using Unity.Collections;
+
+namespace UnityEngine.Formats.Alembic.Importer
+{
+    static class Utils
+    {
+        public static void DisposeIfPossible<T>(this NativeArray<T> array) where T : struct
+        {
+            if (array.IsCreated)
+            {
+                array.Dispose();
+            }
+        }
+
+        public static NativeArray<T> ResizeIfNeeded<T>(this ref NativeArray<T> array, int newLength, Allocator a = Allocator.Persistent) where T : struct
+        {
+            if (array.Length != newLength)
+            {
+                array.DisposeIfPossible();
```

Comment:
unfortunately this is an NativeArray and not a NativeList. Arrays have a fixed size at allocation.
This is merely a refactor to reuse the same "resizing" code instead of copy pasting it.

At some point I need to steal the NativeList code from unity.collections (dots). I cannot use it as is because they are not verified yet.

Reply:
Tanks for the explanation.